kreuzberg/cli.py

```diff
@@ -0,0 +1,130 @@
+import sys
+from pathlib import Path
+from typing import Optional, TextIO
+
+import click
+import tomli
+
+from kreuzberg import ExtractionConfig, ExtractionResult, extract_bytes, extract_file
+from kreuzberg._types import TesseractConfig
+
+
+def load_config(config_path: Optional[Path] = None) -> dict:
```

Comment:
Use

`from future import __annotations__` and update the typing syntax to python 3.10+ syntax